lib/matplotlib/tests/test_collections.py

```diff
@@ -734,7 +734,7 @@ def test_quadmesh_deprecated_signature(
     X += 0.2 * Y
     coords = np.stack([X, Y], axis=-1)
     assert coords.shape == (3, 4, 2)
-    C = np.linspace(0, 2, 12).reshape(3, 4)
+    C = np.linspace(0, 2, 6).reshape(2, 3)
```

Comment:
With the change in #20268 and #20332 (shading's default value being `'flat'`. This data is inconsistent with the behavior that 'flat' accepts 
Cell edges: Z.shape = M, N, len(x) = N+1, len(y) = M+1`